README.md

```diff
@@ -48,6 +48,21 @@ With those installed, setup your computer with:
 
 Cog will be run on `http://localhost:4000`.
 
+## Testing
+
+    $ make test
+
+HTTP requests in integration tests are recorded and stubbed out in
+future test runs. To regenerate these stubs by making actual HTTP
+requests delete the files you wish to regenerate and run the tests.
+
+For instance, to update all tests you could run this:
+
+    $ rm test/fixtures/cassettes/* && make test
+
+Just make sure you have the `TEST_SLACK=1` and `TEST_HIPCHAT=1`
+environment variables set.
+
```

Comment:
Good point. New cassettes are created automatically when using the `use_cassette` macro or `AdapterCase`. I'll include a note about that.